src/app/services/communibee-backend/content/content.ts

```diff
@@ -0,0 +1,12 @@
+import BackendModel from '../backend-model';
+import { Category, CategoryModel } from '../category/category';
+
+export interface Content {
+  title: string;
+  files: string[];
```

Comment:
We've changed the type of `files`, please change accordingly.
https://github.com/CommuniBee/CommuniBee-Back-End/pull/18/files?utf8=%E2%9C%93&diff=split#diff-17340bea4b94c84c1dedf5dd01a8e4fb

Reply:
done, but this changes isn't in the master yet

Reply:
I know, it should have been merged last week, but Ran didn't have time for that. Do you want to review it instead? It will really help.

Reply:
Anyway, the buffer is just the file of whatever type it is. The backend is not aware of which type it is, it is just the file's content.